doc/deploy_starrocks_with_operator_howto.md

```diff
@@ -0,0 +1,297 @@
+> problem.
+
+## 3. Manage StarRocks Cluster
+
+### 3.1. Access StarRocks Cluster
+
+The components of the StarRocks cluster can be accessed through their associated Services, such as the FE Service. For
+detailed descriptions of Services and their access addresses,
+see [api.md](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/api.md)
+and [Services](https://kubernetes.io/docs/concepts/services-networking/service/).
+
+```bash
+$ kubectl get svc
+NAME                                 TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)                               AGE
+starrockscluster-sample-fe-search    ClusterIP   None           <none>        9030/TCP                              76s
+starrockscluster-sample-fe-service   ClusterIP   10.96.26.146   <none>        8030/TCP,9020/TCP,9030/TCP,9010/TCP   76s
+```
+
+#### 3.1.1. Access StarRocks Cluster from within Kubernetes Cluster
+
+From within the Kubernetes cluster, the StarRocks cluster can be accessed through the FE Service's ClusterIP.
+
+1. Obtain the internal virtual IP address `CLUSTER-IP` and port `PORT(S)` of the FE Service.
+
+    ```Bash
+    $ kubectl -n starrocks get svc 
+    NAME                                 TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                               AGE
+    starrockscluster-sample-be-search    ClusterIP   None           <none>        9050/TCP                              66s
+    starrockscluster-sample-be-service   ClusterIP   10.96.86.207   <none>        9060/TCP,8040/TCP,9050/TCP,8060/TCP   66s
+    starrockscluster-sample-fe-search    ClusterIP   None           <none>        9030/TCP                              2m27s
+    starrockscluster-sample-fe-service   ClusterIP   10.96.26.146   <none>        8030/TCP,9020/TCP,9030/TCP,9010/TCP   2m27s
+    ```
+
+2. Access the StarRocks cluster by using the MySQL client from within the Kubernetes cluster.
+
+   ```Bash
+   mysql -h 10.100.162.xxx -P 9030 -uroot
+   ```
+
+   Upon deploying a fresh StarRocks cluster, the `root` user's password remains unset, potentially posing a security
+   risk.
+   See [Init Root Password When First Deploy HOWTO](./initialize_root_password_howto.md) for details on how to set the
+   `root` user's password by using helm chart.
+
+#### 3.1.2. Access StarRocks Cluster from outside Kubernetes Cluster
+
+From outside the Kubernetes cluster, you can access the StarRocks cluster through the FE Service's LoadBalancer or
+NodePort. This topic uses LoadBalancer as an example:
+
+1. Run the command `kubectl -n starrocks edit src starrockscluster-sample` to update the StarRocks cluster configuration
+   file, and add `service` field to the `starRocksFeSpec` field.
+
+    ```YAML
+    spec:
+      starRocksFeSpec:
+        service:            
+          type: LoadBalancer # specified as LoadBalancer
+    ```
+
+2. Obtain the IP address `EXTERNAL-IP` and port `PORT(S)` that the FE Service exposes to the outside.
```

Comment:
add two more ways to access the service

- kubernetes node port
- kubectl proxy

And do a simple comparison on these 4 different ways and its suitable case.

Reply:
todo